src/StackExchange.Redis/ClusterConfiguration.cs

```diff
@@ -301,6 +301,7 @@ internal ClusterNode(ClusterConfiguration configuration, string raw, EndPoint or
             }
 
             NodeId = parts[0];
+            IsFail = flags.Contains("fail");
```

Comment:
I intentionally left it off because I didn't want to make assumptions especially say across a failed DC link where the client might be able to see a split brained cluster node but the initial connected node may not. Not set on anything though - we could still _have_ it and not make decisions on it. Thoughts?

Reply:
I think PFAIL is meant to be purely informational "hey this cluster node thinks that node's dead but it doesn't KNOW it's dead, make of it what you will". To your point you could have a situation where cluster node 1 think's it's alive and that cluster node 2 is dead and vice versa. So the question then becomes do you make the decision as to what to do with it in the client, or do you leave it to the library's user. I don't believe there's a canonical behavior here so leaving an informational PFAIL here and letting the user decide what to do about it could be useful.

Reply:
Added a `IsPossiblyFail` property so it's accessible!